BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -31,14 +31,19 @@ public struct AccessibilityElement: Element {
     public var accessibilityFrameSize: CGSize?
     public var wrappedElement: Element
 
+    public var accessibilityIncrement: (() -> Void)? = nil
```

Comment:
Can we document what these would be used for? I imagine most folks aren't familiar

Reply:
Sorry to be so nit picky (and I know the rest of the repo doesn't set a good precedent) but could you update the comments to have triple slashes and copy it to decrement, so it shows up as a doc comment? Same for `accessibilityActivate` (you may also need to remove the white space)